expertise/create_dataset.py

```diff
@@ -294,11 +322,21 @@ def get_submissions(self):
         submissions = []
 
         for invitation_id in invitation_ids:
+            # Assume invitation is valid for both APIs, but only 1
+            # will have the associated notes
+            submissions_v1 = list(openreview.tools.iterget_notes(
+                self.openreview_client, invitation=invitation_id))
+
+            submissions.extend(submissions_v1)
             submissions.extend(list(openreview.tools.iterget_notes(
-                self.openreview_client, invitation=invitation_id)))
+                self.openreview_client_v2, invitation=invitation_id)))
 
         if paper_id:
-            submissions.append(self.openreview_client.get_note(paper_id))
+            try:
```

Comment:
Not a good way to use `try-except`. It could fail because of other exceptions as well. Use a separate `try-except` block for both API calls and only catch the exception for NoteNotFound.

Reply:
Also, throw an exception if Note is not found by both API calls.